File: 백준/Bronze/2953. 나는 요리사다/나는 요리사다.cc
#include <iostream>
using namespace std;

int main() {
    int max = 0;
    int score = 0;
    int idx = 0;


    for (int i=1; i<=5; i++){
        int sum = 0;
        for (int j=0; j<4; j++) {
            cin>>score;
            sum += score;
        }
        if (sum > max) {
            max = sum;
            idx = i;
        }
    }
    cout << idx << " " << max;
}

File: 백준/Bronze/2953. 나는 요리사다/나는 요리사다.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int max = 0;
        int idx = 0;

        for (int i = 1; i <= 5; i++) {
            int sum = 0;
            for (int j = 0; j < 4; j++) {
                int score = scanner.nextInt();
                sum += score;
            }
            if (sum > max) {
                max = sum;
                idx = i;
            }
        }

        System.out.println(idx + " " + max);
        scanner.close();
    }
}
